packages/shared/src/components/tabs/TabList.tsx

```diff
@@ -113,7 +118,7 @@ function TabList<T extends string = string>({
         );
 
         return (
-          <button
+          <Tag
```

Comment:
If now is an anchor maybe we want to add useful attributes like `aria-label`, `title`, `href` (this should be required imo). What do you think?

Reply:
I've added label and title. Adding href needs a bit more work I see

Reply:
If it's hard to implement, also we can put `role="button"` and `href="#"` if the behaviour is fully JS-side

Reply:
Had a chat with @rebelchris, we'll merge it without the href and then run it through the SEO tool to look. If it still complains, I'll refactor it so that it gets proper href.